src/main/java/pages/TestTypeDetailsPage.java

```diff
@@ -1,18 +1,140 @@
+    private static final String SEATBEALT_INSTALLATION_CHECK_DONE_BUTTON_ID = "Done";
+    private static final String SEATBEALT_INSTALLATION_CHECK_CANCELED_BUTTON_ID = "Cancel";
+    private static final String MOST_RECENT_INSTALLATION_CHECK_DONE_BUTTON_ID = "Done";
+    private static final String MOST_RECENT_INSTALLATION_CHECK_CANCEL_BUTTON_ID = "Cancel";
+    private static final String SEATBELT_TITLE_ID = "Seatbelt installation check";
+
+    public void selectCarriedOutOption() {
+        findElementByXpath("//XCUIElementTypeButton[starts-with(@name,'Carried out during this test')]").click();
+    }
+
+    public boolean checkIsCarriedOutValue(String yesOrNo) {
+        return findElementByXpath("//XCUIElementTypeButton[contains(@name,'Carried out during this test " + yesOrNo + " arrow forward')]").isDisplayed();
+    }
+
+    public void selectIsCarriedOut() {
+        findElementById(CARRIED_OUT_YES_OPTION_ID).click();
+    }
+
+    public void selectIsNotCarriedOut() {
+        findElementById(CARRIED_OUT_NO_OPTION_ID).click();
+    }
+
+    public void cancelIsCarriedOut() {
+        findElementById(CARRIED_OUT_CANCEL_OPTION_ID).click();
+    }
+
+    public void selectSeatbeltsNumberOption() {
+        findElementByXpath("//XCUIElementTypeButton[starts-with(@name,'Number of seatbelts fitted')]").click();
+    }
+
+    public void setSetbealtNumber(String seatbeltNumber) {
+        WebElement input = findElementByXpath(SEATBEALT_INSTALLATION_CHECK_INPUT_FIELD);
+        input.clear();
+        input.sendKeys(seatbeltNumber);
+    }
+
+    public void doneSeatbeltInstallationCheck() {
+        findElementById(SEATBEALT_INSTALLATION_CHECK_DONE_BUTTON_ID).click();
+    }
+
+    public boolean checkSetbealtNumber(String seatbeltNumber) {
+        return findElementByXpath("//XCUIElementTypeButton[contains(@name,'Number of seatbelts fitted " + seatbeltNumber + "')]").isDisplayed();
+    }
+
+    public boolean isSeatbeltTitleDisplayed() {
+        return findElementById(SEATBELT_TITLE_ID).isDisplayed();
+    }
+
+    public boolean isSeatbetlInputFieldDisplayed() {
+        return findElementById(SEATBEALT_INSTALLATION_CHECK_INPUT_FIELD).isDisplayed();
+    }
+
+    public void cancelSeatbeltInstallationCheck() {
+        findElementById(SEATBEALT_INSTALLATION_CHECK_CANCELED_BUTTON_ID).click();
+    }
+
+    public void selectRecentInstallationCheckOption() {
+        findElementByXpath("//XCUIElementTypeButton[starts-with(@name,'Most recent installation check')]").click();
+    }
+
+    public void cancelMostRecentInstallationCheck() {
+        findElementById(MOST_RECENT_INSTALLATION_CHECK_CANCEL_BUTTON_ID).click();
+    }
+
+    public void doneMostRecentInstallationCheck() {
+        findElementById(MOST_RECENT_INSTALLATION_CHECK_DONE_BUTTON_ID).click();
+    }
+
+    public void setRecentInstallationDateCheck() {
+        findElementByXpath("//XCUIElementTypeOther[@name=\"web dialog\"]/XCUIElementTypeOther[3]").click();
```

Comment:
click on day number

Reply:
For the moment is the only working solutiion